### ISSUE TITLE
[BUG] - transaction calculate-min-required-utxo fails to parse protocol paremeters json

### ISSUE THREAD

**Issue by Isarafanikov** [input-output-hk/cardano-node#4047]
**Internal/External**
*External*   otherwise.

**Area**
*Other*   Any other topic (Delegation, Ranking, ...).

**Summary**
`transaction calculate-min-required-utxo` fails with `Command failed: transaction calculate-min-value  Error: The Alonzo era protocol parameters in use is missing one or more of the following fields: UTxOCostPerWord, CostModels, Prices, MaxTxExUnits, MaxBlockExUnits, MaxValueSize, CollateralPercent, MaxCollateralInputs. Did
 you intend to use an Alonzo era protocol parameters value?` despite all fields being present, and the very same protocol file being accepted for transaction fee calculation.

**Steps to reproduce**
1. Generate protocol file. I've used blockfrost cli to generate it: `npx bcc query protocol-parameters --testnet --out-file ./protocol.json`
2. Try to call `cardano-cli transaction calculate-min-required-utxo`. Specific arguments I've used: `cardano-cli transaction calculate-min-required-utxo --alonzo-era --protocol-params-file ./protocol.json --tx-out addr_test1qpa7rsvksefrwnce02qqleny74rfjkymp07uhxh8cfvh52gy4l4qzsxdd6qjvy3jdedj20z9dn2xqg4u5j0gnj7c7tls93cjxz+10000000+"5 cb07022e87049490f6707a5621a20a1f32ade4d0dd8e843754e57b41.4a756820546f6b656e2032 + 50 cb07022e87049490f6707a5621a20a1f32ade4d0dd8e843754e57b41.4a756820546f6b656e2033"`
3. See error

**Expected behavior**
Minimal value should be calculated and returned

| **System info** | **(please complete the following information):** |
| --- | --- |
| OS Name | Windows |
| OS Version | 10 LTSC v1809 |
| Node version (output of `cardano-node --version`) | cardano-node 1.33.0 - mingw32-x86_64 - ghc-8.10 git rev 0000000000000000000000000000000000000000 |
| CLI version  (output of `cardano-cli --version`) | cardano-cli 1.33.0 - mingw32-x86_64 - ghc-8.10 git rev 797289144ad87d494603f19b301e6dbf93310f81 |

If the git revision is reported to be `0000000000000000000000000000000000000000`, please use `cabal build` instead of `cabal install` to build the binaries because `cabal install` does not record the git revision in the binaries.
I've used prebuilt windows binaries

**Screenshots and attachments**
My protocol file: https://pastebin.com/6rc6XeCa

**Comment by CarlosLopezDeLara** [comment]
is this on Vasil-dev network?

**Comment by Isarafanikov** [comment]
No, blockfrost uses Alonzo testenet as far as I'm aware, and `calculate-min-required-utxo` explictly asked to use alonzo era
